akka-bbb-apps/src/main/scala/org/bigbluebutton/core/running/MeetingActor.scala

```diff
@@ -833,6 +845,23 @@ class MeetingActor(
     endTimedOutBreakoutRooms()
   }
 
+  private def handleCheckPresentationConversions(): MeetingState2x = {
+    val now = Instant.now()
+    val maxDuration = java.time.Duration.ofMinutes(20)
```

Comment:
This value should be synced with the config `maxConversionTime` of `bigbluebutton.properties`.